src/common/Utilities/Util.h

```diff
@@ -21,6 +21,7 @@
 
 #include "Define.h"
 #include "Errors.h"
+#include "Common.h"
```

Comment:
please let's not start with micro-optimization and compiler optimization talks...